### PR TITLE
Move scripts used in CI to the new location

### PR DESCRIPTION
**Pull request by sergejparity**
**!!! DO NOT MERGE BEFORE `v0.9.19` RELEASED !!!**
This PR moves CI related scripts and helper files to the new location ./scripts/ci.
We are performing this to unify structure of all paritytech repositories. Target is to improve management of the CI related stuff.
And little housekeeping of the .gitlab-ci.yml.
Related issue: https://github.com/paritytech/ci_cd/issues/313
Also this is reflected in article: Policies and Processes: [Where do I start my new project?](https://github.com/paritytech/ci_cd/wiki/Policies-and-Processes:-Where-do-I-start-my-new-project%3F)

### CODE REVIEW

**Review by coderobe — APPROVED** [review]

**Review by alvicsam — APPROVED** [review]

**Comment by alvicsam** [comment]
Probably we can also remove `ci/script.sh` file

**Comment by sergejparity** [comment]
> Probably we can also remove `ci/script.sh` file

Yes, definitely. But this will go to another task about travis-ci removal.

**Comment by chevdor** [comment]
I see lots of changes related to the release scripts and I expect that we want the next release to NOT be blocked by potential issues with the refactoring. So I would appreciate if we could wait for 0.9.19 to be out until we merge that.

**Pull request merged**
